ci/pip_requirements.txt

```diff
@@ -3,7 +3,7 @@ coveralls
 natsort
 lockfile
 CacheControl
-Sphinx
+Sphinx <= 3.0
```

Comment:
Can you update [doc/README.md](https://github.com/biocore/scikit-bio/blob/master/doc/README.md)? It currently suggests that you `pip install Sphinx....` but should be updated to `pip install 'Sphinx<=3.0' sphinx-bootstrap-theme`.

Reply:
Sure!